DSharpPlus/Net/Serialization/DiscordComponentJsonConverter.cs

```diff
@@ -48,7 +48,7 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                 ComponentType.ActionRow => new DiscordActionRowComponent(),
                 ComponentType.Button when (int)job["style"] is 5 => new DiscordLinkButtonComponent(),
                 ComponentType.Button => new DiscordButtonComponent(),
-                ComponentType.Select => new DiscordSelectComponent(),
+                ComponentType.StringSelect => new DiscordSelectComponent(),
```

Comment:
should the other select types not also be added here?

Reply:
correct

Reply:
It was a small oversight.